iree/compiler/Conversion/LinalgToSPIRV/CodeGenOptionUtils.h

```diff
@@ -35,6 +35,7 @@ namespace iree_compiler {
 struct SPIRVCodegenOptions {
   llvm::SmallVector<unsigned, 3> workgroupSize = {};
   llvm::SmallVector<unsigned, 3> tileSizes = {};
+  llvm::SmallVector<unsigned, 3> invocationTileSizes = {};
```

Comment:
Nit: it would be nice to rename `tileSizes` at L37 into `workgroupTileSizes` here too.